src/main/java/org/gridsuite/network/map/dto/utils/ElementUtils.java

```diff
@@ -61,9 +61,66 @@ public static ConnectablePositionInfos toMapConnectablePosition(Identifiable<?>
         return builder.build();
     }
 
-    public static CurrentLimitsData toMapDataCurrentLimits(Branch<?> branch, Branch.Side side) {
-        Optional<CurrentLimits> limits = side == Branch.Side.ONE ? branch.getCurrentLimits1() : branch.getCurrentLimits2();
-        return limits.map(l -> toMapDataCurrentLimits(l)).orElse(null);
+    public static Optional<ActivePowerControlInfos> toActivePowerControl(Identifiable<?> identifiable) {
+        ActivePowerControlInfos.ActivePowerControlInfosBuilder builder = ActivePowerControlInfos.builder();
```

Comment:
Move the builder inside the `if` block, it isn't need elsewhere.

Reply:
Done